### ISSUE TITLE
Temporarily opened side panel is hard to close

### ISSUE THREAD

**Issue by sangwoo108** [brave/brave-browser#34085]
Reported https://github.com/brave/reviews/issues/1421#issuecomment-1791478022

```
Ok thanks I'll test it out. In the meantime I think I found another UX bug:

1. In settings, change "Sidebar" to "Never" and disable the sidebar button. 
2. Go to a youtube video and click add to playlist then open in playlist.
3. The sidebar is now open with no way to close it.

You can solve this by going to settings, enabling the sidebar button, clicking it, then disabling it again, but the UX is not very good. I imagine this would annoy the users who have disabled sidebar entirely.

EDIT: looks like you can also close it by clicking the playlist icon in the sidebar but this wasn't obvious - maybe need an X button on the sidebar to close it explicitly
```

```
If user disables side panel location bar button and playlist icon from sidebar, it's difficult to close.
How about bring playlist item back when user add item to playst from omnibox?
```

**Comment by sangwoo108** [comment]
Yes, that was exactly what I was thinking of https://github.com/brave/reviews/issues/1421#issuecomment-1791884877

**Comment by MadhaviSeelam** [comment]
Verification `Passed` using
```
Brave | 1.62.45 Chromium: 119.0.6045.134 (Official Build) nightly (64-bit)
-- | --
Revision | 7853f6ed48cd6ea40b7baf3654b72b57193d32c5
OS | Windows 11 Version 22H2 (Build 22621.2428)
```

1. Installed 1.62.45
2. launched Brave
3. verified `#brave-ai-chat` is enabled as a default in `brave://flags`
4. verified Leo icon is shown on the sidebar as a default`
5. right clicked on Leo button and removed Leo item from sidebar
6. confirmed Leo button is removed 
7. typed any keyword  (crypto) at omnibox 
8. selected Leo omnibox result
9. confirmed Leo sidepanel opened
10. confirmed Leo button is added to the sidebar and shown as activated/highlighted
11. clicked `Accept & begin`
12. confirmed `crypto` keyword in the sidepanel

step 3 | step 4 | step 5| step 6 | step 7 | step 9 | step 12
------------ | --------- | -------- | --------- | ------- | ---------  | -------
![image](https://github.com/brave/brave-browser/assets/98358127/fa2e55f5-bff5-48b7-b562-57bf5b1f33bc)|![image](https://github.com/brave/brave-browser/assets/98358127/66dd7aed-cc99-49a4-9a4a-2624e367213e)|![image](https://github.com/brave/brave-browser/assets/98358127/a60c68ab-5ee3-4d02-becd-5ee2dc3c480d)|![image](https://github.com/brave/brave-browser/assets/98358127/c54009da-ddfe-4d0f-a92a-2c6d08ac44ae)|![image](https://github.com/brave/brave-browser/assets/98358127/bfa1d843-fb10-4ddb-9b0a-63a81b6d6b2d)|![image](https://github.com/brave/brave-browser/assets/98358127/ff0d9929-7400-4498-8009-3ebf584e6b23)|![image](https://github.com/brave/brave-browser/assets/98358127/28f0613e-6fbc-4121-ae9a-ce3e5e77ffcd)
